<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pad</class>
 <widget class="QMainWindow" name="pad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmallPAD</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon_calculator.png</normaloff>icon_calculator.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSession">
    <property name="title">
     <string>Сессия</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionCopy_last"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="actionMinimize_to_tray"/>
    <addaction name="actionSolve_nonlinear_equation"/>
    <addaction name="actionIntegrate_function"/>
    <addaction name="actionPlot_graph"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionTutorial"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionEnter_license_key"/>
   </widget>
   <addaction name="menuSession"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Загрузить...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Копировать                             Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Вставить                                  Ctrl+V</string>
   </property>
  </action>
  <action name="actionCopy_last">
   <property name="text">
    <string>Скопировать последнее</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="actionMinimize_to_tray">
   <property name="text">
    <string>Свернуть в лоток</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionPlot_graph">
   <property name="text">
    <string>Построить график...</string>
   </property>
  </action>
  <action name="actionSolve_nonlinear_equation">
   <property name="text">
    <string>Решить нелинейное уравнение...</string>
   </property>
  </action>
  <action name="actionIntegrate_function">
   <property name="text">
    <string>Проинтегрировать функцию...</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="text">
    <string>Справка по функциям...</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionEnter_license_key">
   <property name="text">
    <string>Ввести ключ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
